include/deal.II/lac/la_parallel_vector.h

```diff
@@ -443,6 +443,15 @@ namespace LinearAlgebra
       void
       swap(Vector<Number, MemorySpace> &v);
 
+      /**
+       * Move assignment operator. Uses the swap method.
```

Comment:
Let's document *what* a function does, not *how* it does that:
```suggestion
       * Move assignment operator.
```